ruby/rails/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.3.1
+
+# install build-tools
+RUN apt-get update -qq \
+  && apt-get install -y \
+       build-essential \
+       libpq-dev \
+       nodejs
```

Comment:
We should get rid of third party dependencies like `jquery` etc... otherwise `nodejs` package is required. For now we'll keep it, but later we're going to provide better examples.